pyproject.toml

```
@@ -83,7 +83,7 @@ dependencies = [
     "python-dateutil",
     "python-dotenv",
     "python-geohash",
-    "pyarrow>=14.0.1, <15",
+    "pyarrow>=14.0.1, <19",
```

Let's set the lower bound to 18
```suggestion
    "pyarrow>=18.1.0, <19",
```